ozaria/site/components/avatar-selector/PageAvatarSelector/index.vue

```diff
@@ -85,35 +134,29 @@ export default Vue.extend({
           <div class="col-xs-8 avatar-grid">
 
             <section class="row">
-
-              <div class="col-xs-4 avatar-item">
-                <div class="avatar2" :class="{selected: selected === 1}" data-avatar="1" @click="handleClick" />
-              </div>
-
-              <div class="col-xs-4 avatar-item">
-                <div class="avatar1" :class="{selected: selected === 2}" data-avatar="2" @click="handleClick" />
-              </div>
-
-              <div class="col-xs-4 avatar-item">
-                <div class="avatar3" :class="{selected: selected === 3}" data-avatar="3" @click="handleClick" />
+              <!-- TODO: COMPUTED FIRST ROW OF AVATAR. COMPUTED SECOND ROW OF AVATAR. -->
+              <div class="col-xs-4 avatar-item" v-for="({ selectionImg }, index) in topRowAvatars" :key="index">
```

Comment:
Could the key be a thang id?  Using an index actually has some edge cases that cause rendering not to occur.  It's best to associate the key with something unique to the entry if possible.